pkg/kubelet/kubelet.go

```diff
@@ -1966,10 +1967,12 @@ func (kl *Kubelet) canAdmitPod(pods []*v1.Pod, pod *v1.Pod) (bool, string, strin
 	attrs := &lifecycle.PodAdmitAttributes{Pod: pod, OtherPods: pods}
 	for _, podAdmitHandler := range kl.admitHandlers {
 		if result := podAdmitHandler.Admit(attrs); !result.Admit {
+			klog.InfoS("Pod is rejected", "admitHandler", reflect.TypeOf(podAdmitHandler), "pod", klog.KObj(pod), "podUID", pod.UID, "result", result)
 			return false, result.Reason, result.Message
 		}
 	}
 
+	klog.InfoS("Pod Admitted", "pod", klog.KObj(pod), "podUID", pod.UID)
```

Comment:
admission rejection should be sporadic, but admission success should be way more common, so I think this should be `V(4)`